src/main/java/org/snomed/quality/validator/mrcm/Constants.java

```diff
@@ -2,16 +2,54 @@
 
 public final class Constants {
 
-	public static final String ERROR_MESSAGE = "Please specify the java arguments after replacing the { } with actual values. {release_package_unzipped_root} {is_stated_only} {release_date} {result_dir}";
+	public static final String ERROR_MESSAGE = "Please specify the java arguments after replacing the { } with actual values. {release_package_unzipped_root} {content_type} {release_date} {result_dir}";
 
 	public static final String RELEASE_PACKAGE_UNZIPPED_ROOT_DIR_HELP_MESSAGE = "{release_package_unzipped_root_dir} is for the release package unzipped file root directory. eg: /Users/Releases/SnomedCT_InternationalRF2_PRODUCTION_20170731T120000Z";
 
-	public static final String IS_STATED_ONLY_HELP_MESSAGE = "{is_stated_only} is to state whether to use stated relationships only. This parameter is optional and the default value is set to true.";
+	public static final String CONTENT_TYPE_HELP_MESSAGE = "{content_type} is used to specify whether to use stated, inferred or both (stated & inferred) relationships. " +
```

Comment:
It would be good to inform users how to config both stated and inferred.  Is it by specifying stated&inferred or using comma separated?

Reply:
Fixed.